src/main/java/com/iprd/fhir/utils/FhirResourceTemplateHelper.java

```diff
@@ -132,6 +154,22 @@ public static Location clinic(String state, String district, String city, String
 		facility.setStatus(LocationStatus.ACTIVE);
 		facility.setMode(LocationMode.INSTANCE);
 		facility.setPhysicalType(facilityPhysicalType);
+		try{
+			Location.LocationPositionComponent position = new Location.LocationPositionComponent();
+			position.setLongitude(Double.parseDouble(longitude));
+			position.setLatitude(Double.parseDouble(latitude));
+			facility.setPosition(position);
+			Extension pluscodeExtension = new Extension();
+			pluscodeExtension.setUrl(EXTENSION_PLUSCODE_URL);
+			StringType pluscodeValue = new StringType(pluscode);
+			pluscodeExtension.setValue(pluscodeValue);
+			facility.addExtension(pluscodeExtension);
+		}catch (Exception e){
+			logger.warn("The provided latitude or longitude value is non-numeric. Clinic Details - ", clinic, district, city, state);
```

Comment:
Catch only the numeric exception for this instead of all "Exception" . Or else there maybe some other exception resulting in this log message being incorrect